### PR TITLE
Makes a human's species visible on examine.

### PR DESCRIPTION
**Pull request by FlattestGuitar**
:cl: FlattestGuitar
tweak: a human's species is now visible on examination
/:cl:

As simple as it gets, a human's (this mean any playable species) species will now be visible on examination on the same conditions as their gender already is.

wetskrell

If the game doesn't refer to a person as "they", it will tell you their gender and species.

nuu

A few species are excluded from this treatment: 
- all monkey types to avoid "neara (420), a neara!"
- Abductors, because nobody knows what the hell they are
- Shadowlings, because they're shadowy and stuff

### CODE REVIEW

File: code/modules/mob/living/carbon/human/examine.dm [hunk 1]
```diff
@@ -52,7 +52,16 @@
...
+	msg += "<EM>[src.name]</EM>"
+
+	var/list/nospecies = list("Abductor", "Shadowling", "Neara", "Monkey", "Stok", "Farwa", "Wolpin") //species that won't show their race no matter what
+
+	if (skipjumpsuit && skipface || (src.species.name in nospecies)) //either obscured or on the nospecies list
+		msg += "!\n"	//omit the species when examining
+	else if (src.species.name == "Slime People") //snowflakey because Slime People are defined as a plural
+		msg += ", a Slime Person!\n"
+	else
+		msg += ", a [src.species]!\n"
```
Comment (PJB3005) [inline code/modules/mob/living/carbon/human/examine.dm]: Should be `\a`

File: code/modules/mob/living/carbon/human/examine.dm [hunk 2]
```diff
@@ -52,7 +52,16 @@
...
 
-	msg += "<EM>[src.name]</EM>!\n"
+	msg += "<EM>[src.name]</EM>"
+
+	var/list/nospecies = list("Abductor", "Shadowling", "Neara", "Monkey", "Stok", "Farwa", "Wolpin") //species that won't show their race no matter what
+
+	if (skipjumpsuit && skipface || (src.species.name in nospecies)) //either obscured or on the nospecies list
+		msg += "!\n"	//omit the species when examining
+	else if (src.species.name == "Slime People") //snowflakey because Slime People are defined as a plural
+		msg += ", a Slime Person!\n"
```
Comment (PJB3005) [inline code/modules/mob/living/carbon/human/examine.dm]: Should add a non plural var to the species datums and use that instead of hardcoding.

**Comment by Regen1** [comment]
Perhaps make it obscured by full covering materials such as hardsuits etc?

**Comment by FlattestGuitar** [comment]
As I said in the PR description, this does go away once you wear anything that obscures the visibility of your jumpsuit and covers your face.

File: code/modules/mob/living/carbon/human/examine.dm (hunk 2 above)
Comment (FlattestGuitar) [inline code/modules/mob/living/carbon/human/examine.dm]: Literally every single race has their plural and singular nouns done right, but Slime People have "Slime People" both under singular and plural.

I'm not sure if changing the name to a proper, singular one will be okay with the rest of the code, especially the the SQL.

File: code/modules/mob/living/carbon/human/examine.dm (hunk 1 above)
Comment (tigercat2000) [inline code/modules/mob/living/carbon/human/examine.dm]: also no src

**Comment by fludd12** [comment]
Darn it, I made this, too - I can't work github to make a PR for it, though. Good work!

**Comment by NTSAM** [comment]
Thank god 👍

**Comment by FlattestGuitar** [comment]
Alright, the species' names are no longer capitalized, because BYOND, but it's not that big of a deal, I guess.

**Comment by KasparoVy** [comment]
A pretty big QoL feature, I say. Thank you!

**Pull request merged**